src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/dto/UpdateLocationRequest.kt

```diff
@@ -20,4 +20,5 @@ interface UpdateLocationRequest {
   val deactivationReason: DeactivatedReason?
   val proposedReactivationDate: LocalDate?
   val deactivatedDate: LocalDate?
+  fun isDeactivated() = deactivationReason != null && deactivatedDate != null
```

Comment:
hmm yes this was worrying me a bit, Ive just checked and there are a lot of rows in Nomis where the 3 deactivated/ reactivate columns are null but active is false too. Might need an explicit 'active' field?

Reply:
Or .. I could set the reason as 'OTHER' in these cases?

Reply:
Ah ok, or if not active have a deactivation of OTHER with a last modified date for deactivation?